### PR TITLE
Realign cpugpu

### PR DESCRIPTION
**Pull request by mariadalfonso**
realign GPU-CPU after PR 32146

### CODE REVIEW

**Comment by mariadalfonso** [comment]
Screen Shot 2020-11-21 at 15 23 44

File: RecoLocalCalo/HcalRecProducers/src/MahiGPU.cu [hunk 1]
```diff
@@ -650,11 +650,18 @@ namespace hcal {
...
                                  : 0;
 
-      // store to global
-      pulseMatrix[ipulse * nsamples + sample] = value;
-      ;
-      pulseMatrixM[ipulse * nsamples + sample] = value_t0m;
-      pulseMatrixP[ipulse * nsamples + sample] = value_t0p;
+      if (amplitude > 0.f) {
+        pulseMatrix[ipulse * nsamples + sample] = value;
+        ;
```
Comment (fwyzard) [inline RecoLocalCalo/HcalRecProducers/src/MahiGPU.cu:655]: why the extra `;` here and below ?

Comment (mariadalfonso) [reply RecoLocalCalo/HcalRecProducers/src/MahiGPU.cu]: removed spurious line

**Comment by fwyzard** [comment]
Tested on top of CMSSW_11_2_0_pre9_Patatrack:
image
image

**Comment by mariadalfonso** [comment]
@fwyzard 
You have two plots in  
https://github.com/cms-patatrack/cmssw/pull/576#issuecomment-731862098
To which workflows they correspond ?
Run2 and Run3 ?

**Comment by fwyzard** [comment]
Ciao Maria, the plots are from
https://github.com/cms-patatrack/cmssw/blob/CMSSW_11_2_X_Patatrack/RecoLocalCalo/HcalRecProducers/test/make_GPUvsCPU_HCAL_rechits.py


So, if I understood correctly, it's Run 2 data reconstructed with a Run
3-like configuration ?

**Pull request merged**